third_party/move/tools/move-formatter/tests/formatter/tuple/input1.move.fmt

```diff
@@ -0,0 +1,23 @@
+address 0x42 {
+    module example {
+        // all 3 of these functions are equivalent
+        
+        // when no return type is provided, it is assumed to be `()`
+        fun returs_unit_1() { 
```

Comment:
I am not sure this is the desired format output. For example, rust fmt keeps function bodies that are empty in the same line.

So, this would have been formatted as:
```
        fun returs_unit_1() {}
```

which is more readable than what it is formatted into here.

Reply:
we have updated it according to your suggestions